pact/dao/dao.pact

```diff
@@ -0,0 +1,279 @@
+(enforce-pact-version "3.7")
+
+(namespace (read-msg 'dao-ns))
+
+(module utils-v1 GOVERNANCE
```

Comment:
No need for a separate module. 
- `util.guards` has `chain-time` (now)
- `yesterday`/`tomorrow` are unused
-  block heights are unused, but in any case we should code around dates whenever possible)
The rest should be be folded into `dao-v1` as needed.

Reply:
Also we haven't been naming modules with versions (`dao-v1`), just interfaces (fungible-v1). Governance means module hashes are the real version.

Reply:
Sure, but what do we call it then? `dao.dao` is kinda dumb. `dao.guardians`?

Reply:
fancy name time! `dao.poodles`